src/components/Messages/MessageText/index.js

```diff
@@ -11,7 +12,7 @@ export const MessageText = memo(({
 }) => (
 	<div
 		// eslint-disable-next-line react/no-danger
-		dangerouslySetInnerHTML={{ __html: renderMarkdown(text) }}
+		dangerouslySetInnerHTML={{ __html: emojify(renderMarkdown(text), { output: 'unicode' }) }}
```

Comment:
Why not applying the `emojiify` stuff into `renderMarkdown` method?

Reply:
hmm, that would be better